packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -40,5 +40,14 @@ export const messages = {
 
   client_name: 'Apex Language Server',
   cannot_determine_workspace:
-    'Unable to determine workspace folders for workspace'
+    'Unable to determine workspace folders for workspace',
```

Comment:
@ruthemmanuelle here are the new error messages, let me know if any changes are needed.